main_he.tex

```diff
@@ -0,0 +1,61 @@
+% !TeX spellcheck = en_US
+\documentclass[12pt]{article}
+
+
+% Language setting
+\usepackage{fontspec} % For font settings
+\usepackage{polyglossia} % Better for handling multiple languages with XeLaTeX or LuaLaTeX
+
+% Set languages
+\setmainlanguage{hebrew} % Set the main language to Hebrew
+\setotherlanguage{english} % Optional: if you have English text as well
+
+% Set main font for Latin script (English)
+\setmainfont{Times New Roman}
+
+% Set font for Hebrew script
+\newfontfamily\hebrewfont{Times New Roman}[
+  Script=Hebrew,
+  Scale=MatchLowercase
+]
+
+% Set font for monospace (if needed)
+\newfontfamily\hebrewfonttt{DejaVu Sans Mono}[
+  Script=Hebrew,
+  Scale=MatchLowercase
+]
+
+% If you need to use monospace font for both scripts
+\setmonofont{DejaVu Sans Mono}
+
+
+
+
+\def\sections{sections/translated/he}
+\def\lang_header_adjustment{0px}
+
+\newcommand{\version}{
+  גרסה \textenglish{1.2}
```

Comment:
Are you sure about this? There's a different version handling now.

Reply:
I just copied what I saw when I wrote this, when there will be a new version I'll update it too